### ISSUE TITLE
Missing Seeds in workflows

### ISSUE THREAD

**Issue by aloeliger** [cms-l1t-offline/cmssw#1136]
### What

We have been notified that several seeds seem to be missing in standard relval workflows, specifically `L1_SingleJet60er2p5` and `L1_SingleJet60_FWD3p0`. These are hardcoded as a part of the seeds checked for uGT timing for prescaled seeds [here](https://github.com/cms-sw/cmssw/blob/609dc7ca279d7c1eef5305dac1ff549b63039496/DQM/L1TMonitor/python/L1TStage2uGTTiming_cfi.py#L25-L42). We should recommend a replacement or removal of these seeds.

### Recipe to recreate

Run 2023 relval 141.008

**Comment by eyigitba** [comment]
Hi @aloeliger , I think we should do the following replacements:


```
Remove:
L1_SingleJet60
L1_SingleJet60er2p5
L1_SingleJet60_FWD3p0

Add:
L1_SingleJet35
L1_SingleJet35er2p5
L1_SingleJet35_FWD2p5
```

@lathomas @caruta can you confirm this looks good to you?

**Comment by aloeliger** [comment]
@lathomas, @cartua I would like to start cleaning up the L1 logs which are notably verbose right now. Unless you have any major objections, I will try to commit Efe's suggestion at some point today.

**Comment by lathomas** [comment]
Sounds good for me, thanks.

**Comment by caruta** [comment]
Hi, I'm fine with it too. Thanks!

**Comment by aloeliger** [comment]
@eyigitba @lathomas @caruta, Apparently `L1_ETT1200` is also missing. Do we have a suitable replacement for that?

**Comment by caruta** [comment]
Hi @aloeliger, we're going to remove L1_ETT1200 starting from the next L1 menu: 2023_v1_2_0, which will be deployed before the end of this month. FYI, [here](https://github.com/cms-l1-dpg/L1MenuRun3/tree/master/development/L1Menu_Collisions2023_v1_2_0#removal-of-unused-seeds-cmslitdpg-1155) you can find the complete list of seeds that will be removed in the next menu.
I think we can replace L1_ETT1200 with L1_ETT1600: @lathomas, @eyigitba do you agree?

**Comment by eyigitba** [comment]
Yes, I think that should be fine

**Comment by aloeliger** [comment]
Closed in cms-sw/cmssw#41972
